pkg/datastore/mysql/ensurer/indexes.sql

```diff
@@ -78,6 +78,10 @@ CREATE INDEX deployment_status_updated_at_desc ON Deployment (Status, UpdatedAt
 ALTER TABLE Deployment ADD COLUMN PipedId VARCHAR(36) GENERATED ALWAYS AS (data->>"$.piped_id") VIRTUAL NOT NULL;
 CREATE INDEX deployment_piped_id ON Deployment (PipedId);
 
+-- index on `CompletedAt` DESC
```

Comment:
nits, this should be on `CompletedAt` and `Id`

Reply:
Thank you. I updated, PTAL